Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -253,6 +253,10 @@ Consume the S3 Storage, in two different ways:
 If encryption or compression on the wire is needed, specify the `--v2-port-enable` flag.
 If the v2 address type is present in the `ceph quorum_status`, then the output of 'ceph mon data' i.e, `ROOK_EXTERNAL_CEPH_MON_DATA` will use the v2 port(`3300`).
 
+### NFS storage
```

Comment:
What is the scenario for NFS in an external cluster? Did a user request it? I don't see much detail in #13277. Seems like they would typically just use CephFS.

Reply:
yes there was a request from multiple upstream and downstream users